app/main.py

```diff
@@ -4,0 +1,85 @@
+import datetime
+from app.claasss import Customer, Shops
+from app.fiilee import Config
+
+
+def shop_trip() -> None:
+    config = Config()
+    res_shop = []
+    for elem in config.shops:
+        res_shop.append(Shops(elem["name"],
+                              elem["location"],
+                              elem["products"]))
+
+    res_customer = []
+    for elem in config.customers:
+        res_customer.append(Customer(elem["name"],
+                                     elem["product_cart"],
+                                     elem["location"],
+                                     elem["money"],
+                                     elem["car"]))
+
+    price_for_fuel = config.fuel_price
+    for elem in res_customer:
+
+        print(f"{elem.name} has {elem.money} dollars")
+        res = {}
+        for shop in res_shop:
+            distance = ((elem.location[0] - shop.location[0]) ** 2
+                        + (elem.location[1] - shop.location[1]) ** 2) ** 0.5
+            liters_needed = (distance / 100) * elem.car["fuel_consumption"]
+            price = liters_needed * price_for_fuel
+            money_need = ((elem.product_cart["milk"]
```

Comment:
The code assumes that 'milk', 'bread', and 'butter' are always present in both 'product_cart' and 'shop.products'. Consider adding checks to ensure these keys exist to prevent potential KeyError exceptions.